packages/repack/src/webpack/loaders/assetsLoader/assetsLoader.ts

```diff
@@ -205,9 +229,9 @@ export default async function repackAssetsLoader(this: LoaderContext<Options>) {
             assets,
             assetsDirname,
             remotePublicPath: options.remote.publicPath,
-            resourceDirname,
+            resourceDirname: remoteAssetResource?.path || resourceDirname,
             resourceExtensionType,
-            resourceFilename,
+            resourceFilename: remoteAssetResource?.filename || resourceFilename,
```

Comment:
I think nullish coalescing operator `??` fits better here in both cases, WDYT?

Reply:
I don't have a stron opinion on this one, we can go with `??`. If someone will pass accidentally an empty string, this error will be more visible instead silently falling back to defaults